homeassistant/components/velbus/__init__.py

```diff
@@ -44,7 +45,12 @@ async def velbus_connect_task(
     controller: Velbus, hass: HomeAssistant, entry_id: str
 ) -> None:
     """Task to offload the long running connect."""
-    await controller.connect()
+    try:
+        await controller.connect()
+    except ConnectionError as ex:
+        raise PlatformNotReady(
```

Comment:
This won't work. `PlatformNotReady` can only be raised within a platform setup function. If you want to retry the config entry setup you should raise `ConfigEntryNotReady` inside `async_setup_entry` here in the init module, but not within as task.